index.html

```diff
@@ -244,9 +269,71 @@
         </dt>
       </dl>
       <p>
-        The <code>PermissionName</code> enum defines the list of known
-        permission names.
+        Each enumerator-value in the <a>PermissionName</a> enum identifies a
+        <dfn data-export="">permission</dfn>, which consists of the following
```

Comment:
https://github.com/w3c/permissions/blob/gh-pages/contributing.md says to run tidy, which adds the `=""`. :-/